code/public/index.html

```diff
@@ -13,7 +13,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <title>⟤Task Manager</title>
```

Comment:
I am not sure if it was an Icon or emoji you added here. But it does not show up on the browser. You might want to look into this.